src/panels/config/integrations/integration-panels/zwave_js/zwave_js-node-config.ts

```diff
@@ -0,0 +1,422 @@
+  setNodeConfigParameter,
+  ZWaveJSNodeConfigParams,
+} from "../../../../../data/zwave_js";
+import "../../../../../layouts/hass-tabs-subpage";
+import { haStyle } from "../../../../../resources/styles";
+import type { HomeAssistant, Route } from "../../../../../types";
+import "../../../ha-config-section";
+import { configTabs } from "./zwave_js-config-router";
+import {
+  DeviceRegistryEntry,
+  computeDeviceName,
+  subscribeDeviceRegistry,
+} from "../../../../../data/device_registry";
+import { SubscribeMixin } from "../../../../../mixins/subscribe-mixin";
+import { UnsubscribeFunc } from "home-assistant-js-websocket";
+import memoizeOne from "memoize-one";
+
+const getDevice = memoizeOne(
+  (deviceId: string, entries?: DeviceRegistryEntry[]) =>
+    entries?.find((device) => device.id === deviceId)
+);
+
+const getNodeId = memoizeOne((device: DeviceRegistryEntry) => {
+  const identifier = device.identifiers.find(
+    (ident) => ident[0] === "zwave_js"
+  );
+  if (!identifier) {
+    return undefined;
+  }
+
+  return parseInt(identifier[1].split("-")[1]);
+});
+
+@customElement("zwave_js-node-config")
+class ZWaveJSNodeConfig extends SubscribeMixin(LitElement) {
+  @property({ attribute: false }) public hass!: HomeAssistant;
+
+  @property({ attribute: false }) public route!: Route;
+
+  @property({ type: Boolean }) public narrow!: boolean;
+
+  @property({ type: Boolean }) public isWide!: boolean;
+
+  @property() public configEntryId?: string;
+
+  @property() public deviceId!: string;
+
+  @property({ type: Array })
+  private _deviceRegistryEntries?: DeviceRegistryEntry[];
+
+  @internalProperty() private _config?: ZWaveJSNodeConfigParams[];
+
+  @internalProperty() private _error?: string;
+
+  public connectedCallback(): void {
+    super.connectedCallback();
+    this.deviceId = this.route.path.substr(1);
+  }
+
+  public hassSubscribe(): UnsubscribeFunc[] {
+    return [
+      subscribeDeviceRegistry(this.hass.connection, (entries) => {
+        this._deviceRegistryEntries = entries;
+      }),
+    ];
+  }
+
+  protected updated(changedProps: PropertyValues): void {
+    if (!this._config && changedProps.has("_deviceRegistryEntries")) {
+      this._fetchData();
+    }
+  }
+
+  protected render(): TemplateResult {
+    if (this._error) {
+      return html`<hass-error-screen
+        .hass=${this.hass}
+        .error=${this.hass.localize(
+          `ui.panel.config.zwave_js.node_config.error_${this._error}`
+        )}
+      ></hass-error-screen>`;
+    }
+
+    if (!this._config) {
+      return html`<hass-loading-screen></hass-loading-screen>`;
+    }
+
+    const device = this._device!;
+
+    return html`
+      <hass-tabs-subpage
+        .hass=${this.hass}
+        .narrow=${this.narrow}
+        .route=${this.route}
+        .tabs=${configTabs}
+      >
+        <mwc-icon-button slot="toolbar-icon" @click=${this._fetchData}>
+          <ha-svg-icon .path=${mdiRefresh}></ha-svg-icon>
+        </mwc-icon-button>
```

Comment:
When would a user need to manually refesh?

Reply:
leftover from when i copied the dashboard page to start this one